### ISSUE TITLE
Grammar attempts to include non-existant 'source.regexp.python.renpy' scope

### ISSUE THREAD

**Issue by lildude** [renpy/language-renpy#16]
👋 I'm the primary GitHub maintainer for [Linguist](https://github.com/github/linguist) which is used for syntax highlighting on GitHub.com.

As part of this syntax highlighting, Linguist and thus GitHub.com, uses this grammar for highlighting Ren'Py files.

Our grammar validator has detected an error with this grammar:

- [ ] repository `vendor/grammars/language-renpy` (from https://github.com/williamd1k0/language-renpy.git) (1 errors)
    - [ ] Missing include in grammar: `source.renpy` (in `grammars/renpy.cson`) attempts to include `source.regexp.python.renpy` but the scope cannot be found

This is caused by: https://github.com/renpy/language-renpy/blob/ea6e3a0ae3aca71bf3c9c6b1b942ae8dca9ea0f0/source/renpy.tmpl.cson#L916-L923

This section needs to be redefined, removed or the scope added to the grammar.

**Comment by williamd1k0** [comment]
Sorry for the delay.
This was introduced in https://github.com/renpy/language-renpy/commit/9ac1da36fd822b4739a0a2e095cc9a2c9a76a99f

@lildude Does changing it to `source.regexp.python` solve the problem?
Or I need to include a copy of `regular expressions (python).cson` from [atom/language-python](https://github.com/atom/language-python)?

**Comment by lildude** [comment]
> Does changing it to `source.regexp.python` solve the problem?

Yes as Linguist will include the file from the Python grammar that we ship.

> Or I need to include a copy of `regular expressions (python).cson` from [atom/language-python](https://github.com/atom/language-python)?

This would also work, but you'd need to change the scope within it to match `source.regexp.python.renpy` at https://github.com/atom/language-python/blob/master/grammars/regular%20expressions%20(python).cson#L2

Of the two, I'd recommend the former. That way you don't need to keep maintaining a copy and Renpy will continue to gain from the enhancements to that grammar as it evolves within the upstream grammar.  I'd only recommend the latter if you plan to implement renpy-specific changes to it.

**Comment by williamd1k0** [comment]
>Does changing it to source.regexp.python solve the problem?
>>Yes as Linguist will include the file from the Python grammar that we ship.

I believe Ren'Py won't need an alternative regex scope, so I just renamed it to `source.regexp.python` in https://github.com/renpy/language-renpy/commit/c5c17773e5995eabbee40eb0afb0a3953ba908a9.

Thanks for helping! 👋
